addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -13,10 +35,10 @@ options.registry.WebsiteEvent.include({
      */
     allowRoomCreation(previewMode, widgetValue, params) {
         this._rpc({
-            model: this.modelName,
+            model: "event.event",
             method: 'write',
             args: [[this.eventId], {
-                meeting_room_allow_creation: widgetValue
+                meeting_room_allow_creation: widgetValue !== "",
```

Comment:
Only two values expected, but I added a falsy check, just in case. ![image](https://github.com/odoo/odoo/assets/18171034/0abbbb94-e7b9-46dc-b2ff-e159beed527f)